src/Elastic.Apm/Api/ITransaction.cs

```diff
@@ -2,11 +2,17 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Threading.Tasks;
+using Elastic.Apm.Model.Payload;
 
 namespace Elastic.Apm.Api
 {
 	public interface ITransaction
 	{
+		/// <summary>
+		/// Any arbitrary contextual information regarding the event, captured by the agent, optionally provided by the user.
```

Comment:
I think "arbitrary contextual information" isn't correct... this is not arbitrary information. Did you see this description of `Request` somewhere? I'd simply use the description from the [json spec](https://github.com/elastic/apm-server/blob/master/docs/spec/request.json#L4)

Reply:
fixed